projects/1.20/assets/doggy-talents-next/doggytalents/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+    "item.doggytalents.onigiri": "饭团",
+    "item.doggytalents.onigiri.description": "也叫做大饭团",
+    "item.doggytalents.sausage": "香肠",
+    "item.doggytalents.baker_hat": "面包师帽",
+    "item.doggytalents.baker_hat.description": "热腾腾的达克斯犬面包出炉！",
+    "item.doggytalents.chef_hat": "厨师帽",
+    "item.doggytalents.chef_hat.description": "终于有好寿司了......",
+    "item.doggytalents.lab_coat": "实验袍",
+    "item.doggytalents.ceremonial_garb": "祭典礼服",
+    "item.doggytalents.doggy_contacts": "狗狗美瞳",
+    "item.doggytalents.doggy_contacts.description": "用纸+染料染色另一只镜片",
+    "item.doggytalents.dragon_costume_head": "末影龙服饰：头部",
+    "item.doggytalents.dragon_costume_head.description": "闻起来像龙息......",
+    "item.doggytalents.dragon_costume_suit": "末影龙服饰：套装",
+    "item.doggytalents.dragon_costume_suit.description": "为什么用龙蛋制作？有些问题最好别问......",
+    "item.doggytalents.dragon_costume_wings": "末影龙服饰：翅膀",
+    "item.doggytalents.dragon_costume_wings.description": "来个桶滚！",
+
+    "item.doggytalents.christmas_hat": "圣诞帽",
+    "item.doggytalents.christmas_hat.description": "圣诞快乐！",
+    "item.doggytalents.dog_christmas_tree": "圣诞树服饰",
+    "item.doggytalents.dog_christmas_star": "圣诞星",
+    "item.doggytalents.dog_gift_costume": "礼物服饰",
+    "item.doggytalents.dog_gift_costume.description": "任何场合的最佳礼物！用纸+染料添加副色",
+
+    "item.doggytalents.locator_orb_chi": "智之玉",
+    "item.doggytalents.locator_orb_chi.description": "智慧与知识的宝珠",
+    "item.doggytalents.locator_orb_chu": "忠之玉",
+    "item.doggytalents.locator_orb_chu.description": "忠诚与真诚的宝珠",
+    "item.doggytalents.locator_orb_ko": "孝之玉",
+    "item.doggytalents.locator_orb_ko.description": "孝顺的宝珠",
+    "item.doggytalents.locator_orb_gi": "义之玉",
+    "item.doggytalents.locator_orb_gi.description": "正义与道德的宝珠",
+    "item.doggytalents.locator_orb_tei": "悌之玉",
+    "item.doggytalents.locator_orb_tei.description": "兄弟友爱的宝珠", 
+    "item.doggytalents.locator_orb_rei": "礼之玉",
+    "item.doggytalents.locator_orb_rei.description": "礼节与尊重的宝珠",
+    "item.doggytalents.locator_orb_shin": "信之玉",
+    "item.doggytalents.locator_orb_shin.description": "信赖与信用的宝珠",
+    "item.doggytalents.locator_orb_jin": "仁之玉",
+    "item.doggytalents.locator_orb_jin.description": "仁爱与仁慈的宝珠",
+    "item.doggytalents.locator_orb_ka": "火之玉",
+    "item.doggytalents.locator_orb_ka.description": "火焰的宝珠",
+    "item.doggytalents.locator_orb_sui": "水之玉",
+    "item.doggytalents.locator_orb_sui.description": "水流的宝珠",
+    "item.doggytalents.locator_orb_moku": "木之玉",
+    "item.doggytalents.locator_orb_moku.description": "植物之玉",
+    "item.doggytalents.locator_orb_dyable": "可染色定位珠",
+    "item.doggytalents.striped_scarf": "条纹围巾",
+    "item.doggytalents.head_band_blank": "头巾",
+    "item.doggytalents.head_band_mystery": "头巾 - 神秘",
+    "item.doggytalents.head_band_mystery.description": "那个狼系男孩是谁？",
+    "item.doggytalents.head_band_highhh": "头巾 - 嗨嗨嗨~",
+    "item.doggytalents.head_band_highhh.description": "开发者最爱的颜文字",
```

```suggestion
    "item.doggytalents.head_band_highhh.description": "开发者最爱的emoji",
```
emoji和颜文字是两套组成，emoji应该是黄豆表情为主的一套符号，例如“🤡，😀”，颜文字是用字符组成的表情，例如“:>，：）”。